README.md

```diff
@@ -17,6 +17,7 @@
 
 **Webex Components** is a set of [React](https://reactjs.org) components following Webex standard styling,
 aimed at react developers that want to embed the components into their applications.
+A hosted Storybook containing all of the supported components can be found [here](https://webex.github.io/components/).
```

Comment:
Looking for feedback on this statement.

Reply:
I think it might be better to have a "Demo" section with this sentence in it.